bazel/src/test/data/googleapis/google/example/library/v1/BUILD.bazel.updated.baseline

```diff
@@ -346,6 +346,7 @@ load(
 
 csharp_proto_library(
     name = "library_csharp_proto",
+    extra_opts = ["base_namespace=Google.Cloud.AIPlatform.V1"],
```

Comment:
We will need to add this attribute to the template file(s). Something like the following:
```
  extra_opts = [{{csharp_extra_opts}}],
```
Like what was done [here](https://github.com/googleapis/rules_gapic/pull/155/files#diff-110356d15249484aaf63d441fe58998f27d25a0832ac6172e4aa2a023bba89c2R209)

Then you will also need to add the code to populate the "template data" with the attribute `"csharp_extra_opts"`, like what was done [here](https://github.com/googleapis/rules_gapic/pull/155/files#diff-30b30d6d98bc8b7485b1850bc3260a7a35553b5b7ceecef71e1c891c39bd00dbR46)

All of this means that we will have to generate an empty `extra_opts` attribute if it was not set at all :/

Reply:
> All of this means that we will have to generate an empty extra_opts attribute if it was not set at all :/

Urgh. Ah well... I don't think it'll do any harm. Thanks for the nudge - will have a look now...

Reply:
Okay, I think this is now ready. It required a certain amount of guesswork, as the order of attributes in the mustache-not-mustache file wasn't honored in some cases, but the tests now pass...